dataactbroker/handlers/fileHandler.py

```diff
@@ -932,11 +929,9 @@ def list_submissions(self, page, limit, certified):
         submission_details = []
 
         for submission in submissions:
-            job_ids = self.interfaces.jobDb.getJobsBySubmission(submission.submission_id)
-            total_size = 0
-            for job_id in job_ids:
-                file_size = self.interfaces.jobDb.getFileSize(job_id)
-                total_size += file_size or 0
+            total_size = sess.query(func.sum(Job.file_size)).\
+                filter_by(submission_id=submission.submission_id).\
+                scalar()
```

Comment:
Yeah, I was surprised when I tested this on a set of jobs with some `NULL` file sizes and got an actual number . Will update to handle the case where the query doesn't return any rows at all.